manifests/init.pp

```diff
@@ -4,8 +4,11 @@
 # menuing system with syslinux and some pxe tools installed.
 #
 class pxe (
-  $tftp_root='/srv/tftp'
-){
-  include pxe::syslinux
+  $tftp_root        = $pxe::params::tftp_root,
+  $syslinux_archive = undef
+) inherits pxe::params {
+  class { 'pxe::syslinux':
```

Comment:
Yes, sure. I can do that.

BTW: it will be possible that I would also add a small change to support the "IPAPPEND" kernel option besides "APPEND" (used for pxe-boot with multiple interfaces). I will create a PR once the diff is available..